webapp/src/components/layout/search.tsx

```diff
@@ -38,10 +42,15 @@ export function GlobalSearch() {
     setRedirectVal(null);
   }
 
+  const className = clsx(
+    isFocused && styles.form__focus,
+    !isFocused && styles.form__blur
```

Comment:
The constructs like this can be replaced with a ternary operator such as `isFocused ? styles.form__focus : styles.form__blur`.